contracts/libraries/LibTypes.sol

```diff
@@ -36,4 +36,10 @@ library LibTypes {
         uint256 freeBond;
         uint256 cursedBond;
     }
+
+    struct Signature {
```

Comment:
Think we're going to use this struct outside of thirdpartyfacet? If no, let's move it in there